ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py

```diff
@@ -0,0 +1,1024 @@
+#!/usr/bin/python
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# author alok ranjan (alok.ranjan2@hpe.com)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+author:
+  - Alok Ranjan (@ar-india)
+description:
+  - Collect information from a HPE Nimble Storage array running the NimOs operating system. By default, the module will collect basic
```

Comment:
Nimble Operating System or NimOS are preferred. However you can just say "Collect information from a HPE Nimble Storage array" as NimOS is implicit.

Reply:
ok.